src/legacy/core_plugins/interpreter/public/functions/vis_dimension.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const visDimension = () => ({
+  name: 'vis_dimension',
```

Comment:
Quick note on the function name, we've avoided using underscores and other characters in Canvas because it makes the expression much more confusing. It's true for arguments, but it's *especially* true for function names.

For example, if I know I need the "vis dimension" function, I could type it as `visdimension` or `visDimension`, or even `ViSdImEnSiOn`... it's all case-insensitive for that reason. That underscore breaks that convention, now I need to know your naming scheme.